src/data/recipes.js

```diff
@@ -56,6 +56,20 @@ export const butter = itemify({
   recipeType: recipeType.KITCHEN,
 })
 
+/**
+ * @property farmhand.module:recipes.sunButter
+ * @type {farmhand.recipe}
+ */
+export const sunButter = itemify({
+  id: 'sun-butter',
+  name: 'SunButter',
```

Comment:
Looks like we need a space here:

```suggestion
  name: 'Sun Butter',
```